exdb/tests/browser_tests.py

```diff
@@ -308,6 +337,36 @@ def test_load(self):
         self.assertEqual(self.driver.find_element(By.XPATH, '//h2').text, _('Hello user'))
 
 
+class EditExperienceBrowserTest(DefaultLiveServerTestCase):
+
+    def test_confirm_if_draft_delete(self):
+        e = self.create_experience('dr',
+                                   start=make_aware(datetime(2020, 1, 1, 1, 30), timezone=utc),
+                                   end=make_aware(datetime(2021, 1, 1, 1, 30), timezone=utc))
+        self.client.get(reverse('edit', args=[e.pk]))
+        d = self.driver.find_element(By.CSS_SELECTOR, '#delete')
+        d.click()
```

Comment:
I am copying the logic from ExperienceApprovalBrowserTest to this test as well in the next commit.